docs/en/examples/elementalarea_with_dataobject.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Using ElementalArea with DataObject
+summary: Learn how to add ElementalArea to your models and manage it in a ModalAdmin
+---
+
```

Comment:
```suggestion
```
Title and summary are used in the docs in framework, because those get displayed on silverstripe.org. This isn't the case for this module (e.g. see https://raw.githubusercontent.com/silverstripe/silverstripe-elemental/4/docs/en/searching-blocks.md)